content/docs/2.8/scalers/prometheus.md

```diff
@@ -33,6 +34,7 @@ triggers:
 - `threshold` - Value to start scaling for.
 - `namespace` - A namespace that should be used for namespaced queries. These are required by some highly available Prometheus setups, such as [Thanos](https://thanos.io). (Optional)
 - `cortexOrgId` - The `X-Scope-OrgID` header to query multi tenant [Cortex](https://cortexmetrics.io/). (Optional)
+- `ignoreNullValues` - Default is `enable`, set to `disable` to reporting error when prometheus target is lost. (Optional)
```

Comment:
Please, follow [the pattern for documenting parameters](https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler). Something like: 
- `ignoreNullValues` - Value to reporting error when Prometheus target is lost (Values: `true`, `false`, Default: `true`, Optional)
WDYT? (true/false or enable/disable depends on the decision it core repo)

Reply:
Updated. In core repo I use enable/disable cuz feels like more legit